pkg/pointerdb/pointerdb.go

```diff
@@ -18,6 +18,9 @@ import (
 	"storj.io/storj/pkg/pointerdb/auth"
 	"storj.io/storj/pkg/storage/meta"
 	"storj.io/storj/storage"
+	"storj.io/storj/pkg/kademlia"
+	"storj.io/storj/pkg/dht"
+	dr "storj.io/storj/pkg/datarepair"
```

Comment:
Use the import path or figure out a better name (e.g. just `repair`). Currently there is no conflict so there shouldn't be need to use a different import name.